src/components/Layout/FullBleed.tsx

```diff
@@ -0,0 +1,16 @@
+import { cloneElement, ReactElement } from "react";
+import { useScrollableParent } from "src/components/Layout/ScrollableParent";
+import { Css, px } from "src/Css";
+
+/** Provides a way to extend the full width of the ScrollableParent */
+export function FullBleed({ children, omitPadding = false }: { children: ReactElement; omitPadding?: boolean }) {
+  const { paddingX } = useScrollableParent();
+  return !paddingX
+    ? children
+    : cloneElement(children, {
+        style: Css.mlPx(paddingX * -1)
+          .w(`calc(100% + ${px(paddingX * 2)})`)
```

Comment:
Just curious, why do we need both negative margin + the width calc?

Reply:
Yes, I honestly didn't even think of doing with with a margin-right value as well. I can update to do it that way instead.
My thought process was that we're moving it to the left x number of pixels, so have it be x number of pixels wider. But you're right, we could use negative margins in both directions to stretch it

Reply:
Updated.